cloud/user/user.go

```diff
@@ -142,7 +142,7 @@ func SelectUser(users []astrocore.User, workspace bool) (astrocore.User, error)
 		Header:         []string{"#", "FULLNAME", "EMAIL", "ID", roleColumn, "CREATE DATE"},
 	}
 
-	fmt.Println("\nPlease select the user who's role you would like to update:")
+	fmt.Println("\nPlease select the user:")
```

Comment:
The user select method is used for more than just updating a users role so this text was misleading. e.g. to add auser to a team this method is invoked.